sql/src/main/antlr/OpenDistroSQLIdentifierParser.g4

```diff
@@ -40,6 +40,7 @@ columnName
 
 alias
     : ident
+    | keywordsCanBeId
```

Comment:
Should we move this `keywordCanBeId` both here and in `qualifiedName` to under `ident`? Otherwise I recall keyword in qualified name with dots will still fail, ex. SELECT a.count ...

Reply:
Good point! Will change